packages/ERTP/src/paymentLedger.js

```diff
@@ -267,12 +266,11 @@ export const makePaymentLedger = (
   /**
    * Used by the purse code to implement purse.deposit
    *
-   * @param {Amount} currentBalance - the current balance of the purse
-   * before a deposit
-   * @param {(newPurseBalance: Amount) => void} updatePurseBalance -
-   * commit the purse balance
+   * @param {Amount} currentBalance - The current balance of the purse before a deposit
```

Comment:
we could disable `jsdocCapitalizeDescription` but I think this default is worth keeping

Reply:
I like it. But it seems to be inconsistently applied. Other places that look identical did not get capitalized.